### ISSUE TITLE
🔗 Loop Carousel - First item spans across entire width on window load (Duplicate of #23735) - [ED-11364]

### ISSUE THREAD

**Issue by dev12-byte** [elementor/elementor#28083]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

When creating a loop carousel and placing it near the top of the page (or anywhere where it is visible on page load) the first loop item gets spanned across the entire widget width while the other loop items are being loaded (presumably). Only after all items are loaded the first item reduces to the correct size. It seems like the other items are being lazy loaded and added to the loop at a later time and the first item tries to fill the gap. 

This issue is unrelated to the active theme (in this case Hello Elementor) and the content of the Loop Grid, although it seems like images in the loop item result in a longer loading time of the other elements.  

https://github.com/user-attachments/assets/e32297f4-f874-4627-8cf2-c3e2f075a5c5

### Steps to reproduce

1. Create a loop Carousel near the top of the page where it will be visible on window load or refresh
2. Add Create a new Template and insert some Text and or images
3. Ensure that at least 2 slides are visible at all times
4. Save and preview the page
5. Refresh the page to see the issue

### Expected behavior

If the items take a second to load the first item should not try to fill the gap. Instead either display the first item in the correct width (widget width/items to show - gap in between) or wait until ALL items are loaded and then display them all at once. 

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server - GPL v8.0.37
	PHP Version: 8.3.9
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 50M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.5.5
	Site URL: https://www.abf.at
	Home URL: https://www.abf.at
	WP Multisite: No
	Max Upload Size: 50 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: de_DE
	Timezone: 1
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.3.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: de-DE
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36

== Active Plugins ==
	301 Redirects
		Version: 2.76
		Author: WebFactory Ltd

	ABF Wordpress Plugin
		Version: 
		Author: 

	Advanced Custom Fields
		Version: 6.3.3
		Author: WP Engine

	Atarim: Visual Website Collaboration, Feedback & Workflow Management
		Version: 4.0.1
		Author: Atarim

	Datareporter Webcare
		Version: 2.1.2
		Author: LEMONTEC

	Download Manager
		Version: 3.2.97
		Author: W3 Eden, Inc.

	DynamicConditions
		Version: 1.6.3
		Author: RTO GmbH

	DynamicTags
		Version: 1.2.2
		Author: RTO GmbH

	Elementor
		Version: 3.22.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.22.1
		Author: Elementor.com

	Enable Media Replace
		Version: 4.1.5
		Author: ShortPixel

	Essential Addons for Elementor
		Version: 5.9.26
		Author: WPDeveloper

	Essential Addons for Elementor - Pro
		Version: 5.8.17
		Author: WPDeveloper

	Interactive Globes
		Version: 1.2.1
		Author: Interactive Globes

	JetFormBuilder
		Version: 3.3.4.1
		Author: Crocoblock

	Matomo Analytics - Ethical Stats. Powerful Insights.
		Version: 5.1.1
		Author: Matomo

	NitroPack
		Version: 1.16.2
		Author: NitroPack Inc.

	OoohBoi Steroids for Elementor
		Version: 2.1.6
		Author: OoohBoi

	Pages with category and tag
		Version: 0.9.0
		Author: YAHMAN

	Real Media Library (Free)
		Version: 4.22.17
		Author: devowl.io

	reCaptcha by BestWebSoft
		Version: 1.75
		Author: BestWebSoft

	UpdraftPlus - Backup/Restore
		Version: 1.24.4
		Author: UpdraftPlus.Com, DavidAnderson

	WP External Links
		Version: 2.61
		Author: WebFactory Ltd

	WP Extra File Types
		Version: 0.5.2
		Author: Davide Airaghi

	WPForms Multilingual
		Version: 0.4.0
		Author: OnTheGoSystems

	WP Mail SMTP
		Version: 4.0.1
		Author: WP Mail SMTP

	WPML Multilingual CMS
		Version: 4.6.11
		Author: OnTheGoSystems

	WPML SEO
		Version: 2.1.0
		Author: OnTheGoSystems

	WPML String Translation
		Version: 3.2.12
		Author: OnTheGoSystems

	Yoast SEO
		Version: 23.0
		Author: Team Yoast


== Elemente Verwendung ==
	
	container : 10
		button : 7
		container : 28
		divider : 1
		eael-divider : 3
		heading : 19
		html : 2
		image : 3
		loop-carousel : 1
		text-editor : 5
	footer : 2
		button : 2
		container : 20
		heading : 11
		image : 1
		loop-grid : 6
		shortcode : 1
		social-icons : 1
		wpml-language-switcher : 2
	header : 3
		button : 120
		container : 75
		eael-advanced-search : 5
		heading : 41
		html : 8
		icon : 10
		image : 4
		shortcode : 2
		spacer : 19
	page : 133
		button : 16
		container : 134
		divider : 9
		eael-adv-accordion : 1
		eael-divider : 5
		eael-svg-draw : 1
		heading : 69
		html : 6
		image : 21
		spacer : 32
		table-of-contents : 1
		template : 4
		text-editor : 36
	popup : 17
		button : 1
		container : 63
		heading : 85
		icon : 3
		image : 2
		jet-form-builder-form : 1
		shortcode : 1
		spacer : 5
		text-editor : 5
	widget : 0
		button : 3
		eael-divider : 1
		spacer : 2
	wp-post : 60
		button : 5
		container : 35
		heading : 14
		html : 4
		image : 11
		spacer : 13
		template : 2
		text-editor : 23
		theme-post-excerpt : 1
	wp-page : 86
		animated-headline : 8
		button : 152
		container : 597
		counter : 4
		divider : 12
		eael-adv-accordion : 10
		eael-advanced-search : 1
		eael-content-timeline : 8
		eael-divider : 29
		eael-logo-carousel : 5
		eael-svg-draw : 13
		eael-team-member-carousel : 1
		flip-box : 48
		heading : 536
		hotspot : 4
		html : 21
		icon : 64
		icon-list : 54
		image : 134
		image-box : 3
		image-carousel : 1
		jet-form-builder-form : 2
		loop-carousel : 4
		loop-grid : 5
		media-carousel : 1
		menu-anchor : 19
		nested-carousel : 2
		share-buttons : 4
		shortcode : 2
		slides : 3
		social-icons : 2
		spacer : 162
		table-of-contents : 2
		template : 6
		testimonial-carousel : 4
		text-editor : 257
		video : 1
	single-post : 3
		container : 5
		heading : 4
		theme-post-content : 2
		theme-post-excerpt : 2
		theme-post-featured-image : 3
		theme-post-title : 2
	loop-item : 13
		button : 2
		container : 26
		heading : 23
		icon : 10
		image : 6
		spacer : 10
		text-editor : 2
		theme-post-featured-image : 2
		theme-post-title : 3
	error-404 : 2
		container : 4
		heading : 4
		icon : 2


== Einstellungen ==
	
	cpt_support: post, page
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1


== Eigenschaften ==
	Custom Fonts: 2
	Custom Icons: 5

== Integrations ==
	
	recaptcha: Aktiv


== Elementor-Experimente ==
	Verbessertes Laden von Assets: Standardmäßig aktiviert
	Verbessertes Laden von CSS: Standardmäßig deaktiviert
	Inline-Schriftarten-Symbole: Standardmäßig deaktiviert
	Additional Custom Breakpoints: Standardmäßig aktiviert
	Flexbox Container: Aktiv
	Grid Container: Standardmäßig aktiviert
	Upgrade Swiper Library: Standardmäßig aktiviert
	Nested Elements Performance: Standardmäßig deaktiviert
	Optimized Control Loading: Inaktiv
	Optimized Markup: Standardmäßig deaktiviert
	Plugin Onboarding: Standardmäßig aktiviert
	Default to New Theme Builder: Standardmäßig aktiviert
	Elementor Home Screen: Standardmäßig aktiviert
	Editor Top Bar: Aktiv
	Build with AI: Inaktiv
	Startseiten: Standardmäßig aktiviert
	Nested Elements: Aktiv
	Hintergrundbilder nachladen: Standardmäßig deaktiviert
	Seiten Panel: Standardmäßig deaktiviert
	Element Caching: Standardmäßig deaktiviert
	Conversion Center: Standardmäßig deaktiviert
	Bedingungen anzeigen: Aktiv
	Form Submissions: Standardmäßig aktiviert
	Menü: Standardmäßig deaktiviert
	Taxonomy Filter: Aktiv


== Protokoll ==
	
JS: showing 16 of 16JS: 2024-06-20 05:48:25 [error X 7][../wp-admin/post.php?post=43958&action=elementor:1:2] "[object Object]" is not valid JSON 
JS: 2024-06-20 05:48:44 [error X 5][../wp-admin/post.php?post=43958&action=elementor:1:2] Unexpected token 'o', "[object Obj"... is not valid JSON 
JS: 2024-06-20 05:51:47 [error X 9142][../wp-content/uploads/essential-addons-elementor/eael.js?ver=1718865402:21:206926] Cannot read properties of undefined (reading 'size') 
JS: 2024-06-21 07:53:38 [error X 1][../wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.22.1:2:16853] Cannot read properties of undefined (reading 'attributes') 
JS: 2024-06-25 09:52:13 [error X 36][../wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.22.1:2:56144] Model.isValidChild() should be implemented, please provide 'isValidChild' functionality. 
JS: 2024-06-26 10:16:19 [error X 2][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.22.1:3:327939] Cannot read properties of undefined (reading 'id') 
JS: 2024-06-26 10:21:47 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view38190") has already been destroyed and cannot be used. 
JS: 2024-06-26 10:22:06 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view38415") has already been destroyed and cannot be used. 
JS: 2024-06-26 10:35:38 [error X 2][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.22.1:3:706805] this.model.isValidChild is not a function 
JS: 2024-06-27 07:40:58 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.22.1:3:657336] Cannot read properties of undefined (reading 'isDesignable') 
JS: 2024-06-28 08:04:43 [error X 7][../wp-includes/js/jquery/jquery.min.js?ver=3.7.1:2:28760] Cannot read properties of undefined (reading 'value') 
JS: 2024-07-01 07:07:43 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.22.1:3:54583] Cannot read properties of undefined (reading 'activate') 
JS: 2024-07-01 07:29:56 [error X 18][../wp-content/uploads/essential-addons-elementor/eael-544.js?ver=1714461676:1:3997] Cannot read properties of undefined (reading 'top') 
JS: 2024-07-01 11:03:49 [error X 2][../wp-content/uploads/essential-addons-elementor/eael.js?ver=1719317774:15:10052] Cannot read properties of null (reading 'r') 
JS: 2024-07-05 08:58:52 [error X 38][../wp-content/plugins/jetformbuilder/assets/js/frontend/main.js?ver=250b7174e5f50c1b4a7a:1:41134] Cannot read properties of undefined (reading 'getName') 
JS: 2024-07-15 06:31:40 [error X 2][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.22.3:3:1176313] Cannot read properties of null (reading 'getBoundingClientRect') 

PHP: showing 16 of 16PHP: 2024-06-20 06:52:38 [error X 2][../wp-content/plugins/elementor-pro/modules/display-conditions/module.php::174] Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, array given in ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php:174
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(174): json_decode(Array, true)
#1 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(181): ElementorPro\Modules\DisplayConditions\Module->get_saved_conditions(Array)
#2 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\DisplayConditions\Module->before_element_render(Object(Elementor\Widget_Icon))
#3 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 ../wp-content/plugins/elementor/includes/base/element-base.php(474): do_action('elementor/front...', Object(Elementor\Widget_Icon))
#6 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#7 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#8 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#10 ../wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#11 ../wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements(Array)
#12 ../wp-content/plugins/elementor/core/base/document.php(1193): Elementor\Core\Base\Document->print_elements(Array)
#13 ../wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper(Array)
#14 ../wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content(43958)
#15 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content('<h2><a href="ht...')
#16 ../wp-includes/plugin.php(205): WP_Hook->apply_filters('<h2><a href="ht...', Array)
#17 ../wp-includes/post-template.php(256): apply_filters('the_content', '<h2><a href="ht...')
#18 ../wp-content/themes/hello-elementor/template-parts/single.php(23): the_content()
#19 ../wp-includes/template.php(812): require('/var/home/abf_a...')
#20 ../wp-includes/template.php(745): load_template('/var/home/abf_a...', false, Array)
#21 ../wp-includes/general-template.php(206): locate_template(Array, true, false, Array)
#22 ../wp-content/themes/hello-elementor/index.php(21): get_template_part('template-parts/...')
#23 ../wp-includes/template-loader.php(106): include('/var/home/abf_a...')
#24 ../wp-blog-header.php(19): require_once('/var/home/abf_a...')
#25 ../index.php(17): require('/var/home/abf_a...')
#26 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-20 06:53:04 [error X 1][../wp-content/plugins/elementor-pro/modules/display-conditions/module.php::174] Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, array given in ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php:174
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(174): json_decode(Array, true)
#1 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(181): ElementorPro\Modules\DisplayConditions\Module->get_saved_conditions(Array)
#2 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\DisplayConditions\Module->before_element_render(Object(Elementor\Widget_Button))
#3 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 ../wp-content/plugins/elementor/includes/base/element-base.php(474): do_action('elementor/front...', Object(Elementor\Widget_Button))
#6 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#7 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#8 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#10 ../wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#11 ../wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements(Array)
#12 ../wp-content/plugins/elementor/core/base/document.php(1193): Elementor\Core\Base\Document->print_elements(Array)
#13 ../wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper(Array)
#14 ../wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content(43958)
#15 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content('<h2><a href="ht...')
#16 ../wp-includes/plugin.php(205): WP_Hook->apply_filters('<h2><a href="ht...', Array)
#17 ../wp-includes/post-template.php(256): apply_filters('the_content', '<h2><a href="ht...')
#18 ../wp-content/themes/hello-elementor/template-parts/single.php(23): the_content()
#19 ../wp-includes/template.php(812): require('/var/home/abf_a...')
#20 ../wp-includes/template.php(745): load_template('/var/home/abf_a...', false, Array)
#21 ../wp-includes/general-template.php(206): locate_template(Array, true, false, Array)
#22 ../wp-content/themes/hello-elementor/index.php(21): get_template_part('template-parts/...')
#23 ../wp-includes/template-loader.php(106): include('/var/home/abf_a...')
#24 ../wp-blog-header.php(19): require_once('/var/home/abf_a...')
#25 ../index.php(17): require('/var/home/abf_a...')
#26 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-20 06:55:28 [error X 1][../wp-content/plugins/elementor-pro/modules/display-conditions/module.php::174] Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, array given in ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php:174
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(174): json_decode(Array, true)
#1 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(181): ElementorPro\Modules\DisplayConditions\Module->get_saved_conditions(Array)
#2 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\DisplayConditions\Module->before_element_render(Object(Elementor\Widget_Heading))
#3 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 ../wp-content/plugins/elementor/includes/base/element-base.php(474): do_action('elementor/front...', Object(Elementor\Widget_Heading))
#6 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#7 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#8 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#10 ../wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#11 ../wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements(Array)
#12 ../wp-content/plugins/elementor/core/base/document.php(1193): Elementor\Core\Base\Document->print_elements(Array)
#13 ../wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper(Array)
#14 ../wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content(43958)
#15 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content('<h2><a href="ht...')
#16 ../wp-includes/plugin.php(205): WP_Hook->apply_filters('<h2><a href="ht...', Array)
#17 ../wp-includes/post-template.php(256): apply_filters('the_content', '<h2><a href="ht...')
#18 ../wp-content/themes/hello-elementor/template-parts/single.php(23): the_content()
#19 ../wp-includes/template.php(812): require('/var/home/abf_a...')
#20 ../wp-includes/template.php(745): load_template('/var/home/abf_a...', false, Array)
#21 ../wp-includes/general-template.php(206): locate_template(Array, true, false, Array)
#22 ../wp-content/themes/hello-elementor/index.php(21): get_template_part('template-parts/...')
#23 ../wp-includes/template-loader.php(106): include('/var/home/abf_a...')
#24 ../wp-blog-header.php(19): require_once('/var/home/abf_a...')
#25 ../index.php(17): require('/var/home/abf_a...')
#26 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-20 06:59:09 [error X 2][../wp-content/plugins/elementor-pro/modules/display-conditions/module.php::174] Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, array given in ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php:174
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(174): json_decode(Array, true)
#1 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(181): ElementorPro\Modules\DisplayConditions\Module->get_saved_conditions(Array)
#2 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\DisplayConditions\Module->before_element_render(Object(Elementor\Widget_Button))
#3 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 ../wp-content/plugins/elementor/includes/base/element-base.php(474): do_action('elementor/front...', Object(Elementor\Widget_Button))
#6 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#7 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#8 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#10 ../wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#11 ../wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements(Array)
#12 ../wp-content/plugins/elementor/core/base/document.php(1193): Elementor\Core\Base\Document->print_elements(Array)
#13 ../wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper(Array)
#14 ../wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content(43958)
#15 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content('<a href="https:...')
#16 ../wp-includes/plugin.php(205): WP_Hook->apply_filters('<a href="https:...', Array)
#17 ../wp-includes/post-template.php(256): apply_filters('the_content', '<a href="https:...')
#18 ../wp-content/themes/hello-elementor/template-parts/single.php(23): the_content()
#19 ../wp-includes/template.php(812): require('/var/home/abf_a...')
#20 ../wp-includes/template.php(745): load_template('/var/home/abf_a...', false, Array)
#21 ../wp-includes/general-template.php(206): locate_template(Array, true, false, Array)
#22 ../wp-content/themes/hello-elementor/index.php(21): get_template_part('template-parts/...')
#23 ../wp-includes/template-loader.php(106): include('/var/home/abf_a...')
#24 ../wp-blog-header.php(19): require_once('/var/home/abf_a...')
#25 ../index.php(17): require('/var/home/abf_a...')
#26 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-20 06:59:59 [error X 1][../wp-content/plugins/elementor-pro/modules/display-conditions/module.php::174] Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, array given in ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php:174
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(174): json_decode(Array, true)
#1 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(181): ElementorPro\Modules\DisplayConditions\Module->get_saved_conditions(Array)
#2 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\DisplayConditions\Module->before_element_render(Object(Elementor\Widget_Button))
#3 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 ../wp-content/plugins/elementor/includes/base/element-base.php(474): do_action('elementor/front...', Object(Elementor\Widget_Button))
#6 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#7 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#8 ../wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#9 ../wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements(Array)
#10 ../wp-content/plugins/elementor/core/base/document.php(1193): Elementor\Core\Base\Document->print_elements(Array)
#11 ../wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper(Array)
#12 ../wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content(43958)
#13 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content('<a href="https:...')
#14 ../wp-includes/plugin.php(205): WP_Hook->apply_filters('<a href="https:...', Array)
#15 ../wp-includes/post-template.php(256): apply_filters('the_content', '<a href="https:...')
#16 ../wp-content/themes/hello-elementor/template-parts/single.php(23): the_content()
#17 ../wp-includes/template.php(812): require('/var/home/abf_a...')
#18 ../wp-includes/template.php(745): load_template('/var/home/abf_a...', false, Array)
#19 ../wp-includes/general-template.php(206): locate_template(Array, true, false, Array)
#20 ../wp-content/themes/hello-elementor/index.php(21): get_template_part('template-parts/...')
#21 ../wp-includes/template-loader.php(106): include('/var/home/abf_a...')
#22 ../wp-blog-header.php(19): require_once('/var/home/abf_a...')
#23 ../index.php(17): require('/var/home/abf_a...')
#24 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-20 07:01:49 [error X 1][../wp-content/plugins/elementor-pro/modules/display-conditions/module.php::174] Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, array given in ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php:174
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(174): json_decode(Array, true)
#1 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(181): ElementorPro\Modules\DisplayConditions\Module->get_saved_conditions(Array)
#2 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\DisplayConditions\Module->before_element_render(Object(Elementor\Includes\Elements\Container))
#3 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 ../wp-content/plugins/elementor/includes/base/element-base.php(474): do_action('elementor/front...', Object(Elementor\Includes\Elements\Container))
#6 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#7 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#8 ../wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#9 ../wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements(Array)
#10 ../wp-content/plugins/elementor/core/base/document.php(1193): Elementor\Core\Base\Document->print_elements(Array)
#11 ../wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper(Array)
#12 ../wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content(43958)
#13 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content('<a href="https:...')
#14 ../wp-includes/plugin.php(205): WP_Hook->apply_filters('<a href="https:...', Array)
#15 ../wp-includes/post-template.php(256): apply_filters('the_content', '<a href="https:...')
#16 ../wp-content/themes/hello-elementor/template-parts/single.php(23): the_content()
#17 ../wp-includes/template.php(812): require('/var/home/abf_a...')
#18 ../wp-includes/template.php(745): load_template('/var/home/abf_a...', false, Array)
#19 ../wp-includes/general-template.php(206): locate_template(Array, true, false, Array)
#20 ../wp-content/themes/hello-elementor/index.php(21): get_template_part('template-parts/...')
#21 ../wp-includes/template-loader.php(106): include('/var/home/abf_a...')
#22 ../wp-blog-header.php(19): require_once('/var/home/abf_a...')
#23 ../index.php(17): require('/var/home/abf_a...')
#24 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-20 07:02:30 [error X 1][../wp-content/plugins/elementor-pro/modules/display-conditions/module.php::174] Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, array given in ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php:174
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(174): json_decode(Array, true)
#1 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(181): ElementorPro\Modules\DisplayConditions\Module->get_saved_conditions(Array)
#2 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\DisplayConditions\Module->before_element_render(Object(Elementor\Includes\Elements\Container))
#3 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 ../wp-content/plugins/elementor/includes/base/element-base.php(474): do_action('elementor/front...', Object(Elementor\Includes\Elements\Container))
#6 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#7 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#8 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#9 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#10 ../wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#11 ../wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements(Array)
#12 ../wp-content/plugins/elementor/core/base/document.php(1193): Elementor\Core\Base\Document->print_elements(Array)
#13 ../wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper(Array)
#14 ../wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content(43958)
#15 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content('<a href="https:...')
#16 ../wp-includes/plugin.php(205): WP_Hook->apply_filters('<a href="https:...', Array)
#17 ../wp-includes/post-template.php(256): apply_filters('the_content', '<a href="https:...')
#18 ../wp-content/themes/hello-elementor/template-parts/single.php(23): the_content()
#19 ../wp-includes/template.php(812): require('/var/home/abf_a...')
#20 ../wp-includes/template.php(745): load_template('/var/home/abf_a...', false, Array)
#21 ../wp-includes/general-template.php(206): locate_template(Array, true, false, Array)
#22 ../wp-content/themes/hello-elementor/index.php(21): get_template_part('template-parts/...')
#23 ../wp-includes/template-loader.php(106): include('/var/home/abf_a...')
#24 ../wp-blog-header.php(19): require_once('/var/home/abf_a...')
#25 ../index.php(17): require('/var/home/abf_a...')
#26 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-20 07:07:18 [error X 4][../wp-content/plugins/elementor-pro/modules/display-conditions/module.php::174] Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, array given in ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php:174
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(174): json_decode(Array, true)
#1 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(181): ElementorPro\Modules\DisplayConditions\Module->get_saved_conditions(Array)
#2 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\DisplayConditions\Module->before_element_render(Object(Elementor\Includes\Elements\Container))
#3 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 ../wp-content/plugins/elementor/includes/base/element-base.php(474): do_action('elementor/front...', Object(Elementor\Includes\Elements\Container))
#6 ../wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#7 ../wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements(Array)
#8 ../wp-content/plugins/elementor/core/base/document.php(1193): Elementor\Core\Base\Document->print_elements(Array)
#9 ../wp-content/plugins/elementor/includes/frontend.php(1157): Elementor\Core\Base\Document->print_elements_with_wrapper(Array)
#10 ../wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content(43958)
#11 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content('<a href="https:...')
#12 ../wp-includes/plugin.php(205): WP_Hook->apply_filters('<a href="https:...', Array)
#13 ../wp-includes/post-template.php(256): apply_filters('the_content', '<a href="https:...')
#14 ../wp-content/themes/hello-elementor/template-parts/single.php(23): the_content()
#15 ../wp-includes/template.php(812): require('/var/home/abf_a...')
#16 ../wp-includes/template.php(745): load_template('/var/home/abf_a...', false, Array)
#17 ../wp-includes/general-template.php(206): locate_template(Array, true, false, Array)
#18 ../wp-content/themes/hello-elementor/index.php(21): get_template_part('template-parts/...')
#19 ../wp-includes/template-loader.php(106): include('/var/home/abf_a...')
#20 ../wp-blog-header.php(19): require_once('/var/home/abf_a...')
#21 ../index.php(17): require('/var/home/abf_a...')
#22 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-21 06:48:00 [warning X 88][../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2024-06-21 06:48:05 [notice X 16][../wp-content/plugins/elementor-pro/modules/screenshots/module.php::33] Function utf8_decode() is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-21 08:21:56 [error X 4][../wp-content/plugins/elementor-pro/modules/display-conditions/module.php::174] Uncaught TypeError: json_decode(): Argument #1 ($json) must be of type string, array given in ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php:174
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(174): json_decode(Array, true)
#1 ../wp-content/plugins/elementor-pro/modules/display-conditions/module.php(181): ElementorPro\Modules\DisplayConditions\Module->get_saved_conditions(Array)
#2 ../wp-includes/class-wp-hook.php(324): ElementorPro\Modules\DisplayConditions\Module->before_element_render(Object(Elementor\Includes\Elements\Container))
#3 ../wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#4 ../wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 ../wp-content/plugins/elementor/includes/base/element-base.php(474): do_action('elementor/front...', Object(Elementor\Includes\Elements\Container))
#6 ../wp-content/plugins/elementor/includes/base/element-base.php(1422): Elementor\Element_Base->print_element()
#7 ../wp-content/plugins/elementor/includes/base/element-base.php(483): Elementor\Element_Base->print_content()
#8 ../wp-content/plugins/elementor/core/base/document.php(1867): Elementor\Element_Base->print_element()
#9 ../wp-content/plugins/elementor/core/base/document.php(1792): Elementor\Core\Base\Document->do_print_elements(Array)
#10 ../wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(412): Elementor\Core\Base\Document->print_elements(Array)
#11 ../wp-content/plugins/elementor/includes/frontend.php(1157): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document->print_elements_with_wrapper(Array)
#12 ../wp-content/plugins/elementor/includes/frontend.php(1052): Elementor\Frontend->get_builder_content(43964)
#13 ../wp-includes/class-wp-hook.php(324): Elementor\Frontend->apply_builder_in_content('<h2>The current...')
#14 ../wp-includes/plugin.php(205): WP_Hook->apply_filters('<h2>The current...', Array)
#15 ../wp-includes/post-template.php(256): apply_filters('the_content', '<h2>The current...')
#16 ../wp-content/plugins/elementor/modules/page-templates/module.php(190): the_content()
#17 [internal function]: Elementor\Modules\PageTemplates\Module->print_callback()
#18 ../wp-content/plugins/elementor/modules/page-templates/module.php(207): call_user_func(Array)
#19 ../wp-content/plugins/elementor/modules/page-templates/templates/canvas.php(38): Elementor\Modules\PageTemplates\Module->print_content()
#20 ../wp-includes/template-loader.php(106): include('/var/home/abf_a...')
#21 ../wp-blog-header.php(19): require_once('/var/home/abf_a...')
#22 ../index.php(17): require('/var/home/abf_a...')
#23 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-06-21 09:34:44 [warning X 6][../wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-image.php::40] Undefined array key 0 [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-image.php(40): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor/core/dynamic-tags/data-tag.php(44): ElementorPro\Modules\DynamicTags\ACF\Tags\ACF_Image -> get_value()
#2: ../wp-content/plugins/elementor/core/dynamic-tags/manager.php(216): Elementor\Core\DynamicTags\Data_Tag -> get_content()
#3: Elementor\Core\DynamicTags\Manager -> get_tag_data_content()
#4: ../wp-content/plugins/elementor/core/dynamic-tags/manager.php(109): class type call_user_func_array()
',
)]
PHP: 2024-06-21 09:34:44 [warning X 6][../wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-image.php::40] Undefined array key 1 [array (
  'trace' => '
#0: ../wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-image.php(40): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ../wp-content/plugins/elementor/core/dynamic-tags/data-tag.php(44): ElementorPro\Modules\DynamicTags\ACF\Tags\ACF_Image -> get_value()
#2: ../wp-content/plugins/elementor/core/dynamic-tags/manager.php(216): Elementor\Core\DynamicTags\Data_Tag -> get_content()
#3: Elementor\Core\DynamicTags\Manager -> get_tag_data_content()
#4: ../wp-content/plugins/elementor/core/dynamic-tags/manager.php(109): class type call_user_func_array()
',
)]
PHP: 2024-07-01 10:53:36 [warning X 461][../wp-content/plugins/elementor-pro/modules/display-conditions/conditions/page-parent-condition.php::37] Attempt to read property "post_parent" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-07-08 10:05:18 [error X 2][../wp-content/plugins/elementor/core/base/base-object.php::109] Maximum execution time of 30 seconds exceeded [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-07-15 07:40:11 [warning X 3][../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::179] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 10 of 102024-07-15 06:59:59 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.1',
  'to' => '3.22.3',
)]
2024-07-15 07:00:00 [info] elementor::elementor_updater Started 
2024-07-15 07:00:00 [info] Elementor/Upgrades - _on_each_version Start  
2024-07-15 07:00:12 [info] Elementor/Upgrades - _on_each_version Finished 
2024-07-15 07:00:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.1',
  'to' => '3.22.3',
)]
2024-07-15 07:00:36 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.0',
  'to' => '3.22.1',
)]
2024-07-15 07:00:37 [info] elementor-pro::elementor_pro_updater Started 
2024-07-15 07:00:37 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-07-15 07:00:37 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-07-15 07:00:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.0',
  'to' => '3.22.1',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Die Kompatibilität ist nicht angegeben
	Essential Addons for Elementor: Die Kompatibilität ist nicht angegeben
	Essential Addons for Elementor - Pro: Die Kompatibilität ist nicht angegeben
	OoohBoi Steroids for Elementor: Die Kompatibilität ist nicht angegeben

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi @dev12-byte!

There is already a similar issue to this on #23735. The Nested Carousel and Loop Carousel share the same infrastructure.

For this reason, I'm closing this issue and merging it to the other one.

Thank you for contributing and sharing your feedback!

Best regards
